From 3fe48ce2841524656c88643d02308a8a63b262ba Mon Sep 17 00:00:00 2001
From: Bilux <i.bilux@gmail.com>
Date: Tue, 25 Feb 2020 19:34:43 +0100
Subject: [PATCH] Remove CAP_SYS_NICE from surfaceflinger

---
 libcutils/fs_config.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libcutils/fs_config.cpp b/libcutils/fs_config.cpp
index f9fddb6d6e5..6fd67187e51 100644
--- a/libcutils/fs_config.cpp
+++ b/libcutils/fs_config.cpp
@@ -191,8 +191,8 @@ static const struct fs_path_config android_files[] = {
                                               "system/bin/run-as" },
 
     // Support FIFO scheduling mode in SurfaceFlinger.
-    { 00755, AID_SYSTEM,    AID_GRAPHICS,  CAP_MASK_LONG(CAP_SYS_NICE),
-                                              "system/bin/surfaceflinger" },
+    /*{ 00755, AID_SYSTEM,    AID_GRAPHICS,  CAP_MASK_LONG(CAP_SYS_NICE),
+                                              "system/bin/surfaceflinger" },*/
     // generic defaults
     { 00755, AID_ROOT,      AID_ROOT,      0, "bin/*" },
     { 00640, AID_ROOT,      AID_SHELL,     0, "fstab.*" },
